app/uk/gov/hmrc/agentservicesaccount/views/pages/suspend/contact_details.scala.html

```diff
@@ -78,13 +78,14 @@
     @if(form.hasErrors){
         @govukErrorSummary(ErrorSummary(errorList = form.errors.asTextErrorLinks, title = Text(msgs("survey.error.summary"))))
     }
-    <p class="govuk-hint govuk-!-margin-50">@msgs("suspend.contact-details.invite.details.hint")</p>
-    <h1 class="govuk-fieldset__legend--l">@msgs("suspend.contact-details.invite.details.heading")</h1>
+    <p class="govuk-hint govuk-!-font-size-27 govuk-!-static-margin-bottom-0">@msgs("suspend.contact-details.invite.details.hint")</p>
+    <h1 class="govuk-fieldset__legend--l govuk-!-margin-top-0">@msgs("suspend.contact-details.invite.details.heading")</h1>
     <p class="govuk-inset-text govuk-!-margin-bottom-6">@msgs("suspend.contact-details.invite.details.inset")</p>
     @formWithCSRF(action = routes.SuspendedJourneyController.submitContactDetails()) {
         @govukFieldset(Fieldset(
             html = fieldsetHtml
         ))
+        <div class="govuk-!-padding-bottom-5"></div>
```

Comment:
couldn't you just add the padding class to the button? eg.

`@govukButton(Button(content = Text(msgs("common.continue-save")), classes = "govuk-!-padding-top-5"
))`

Reply:
see the attributes in play-frontend-hrmc https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/button/Button.scala

https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukButton.scala.html

Reply:
I tried to however doing this will created padding around the text in the button

Reply:
If you're after spacing use margin instead of padding